addOns/ascanrules/CHANGELOG.md

```diff
@@ -186,3 +186,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
 
+[33]: https://github.com/zaproxy/zap-extensions/releases/ascanrules-v33
```

Comment:
The asterisk at L50 seems to be making everything below italicized. Replace with html entity?

Reply:
&#42;nix (`&#42;`)

Reply:
Actual rendering looks fine: https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrules/CHANGELOG.md

Reply:
Huh, ok just a diff view issue